lte/gateway/deploy/roles/magma/tasks/main.yml

```diff
@@ -45,6 +44,12 @@
     dest: /etc/systemd/system/sctpd.service
   when: full_provision
 
+- name: Copy envoy service file
```

Comment:
We don't need this. This should be automated when installing magma

Reply:
Isn't this required for the vm? f.e above we have it for all other services

Reply:
Hmm the apt install should do the trick If you want to make sure we can have it too